examples/e2e-tests/elm/Generated/Api.elm

```diff
@@ -61,8 +64,8 @@ decodeResponseWithArgs =
     succeed ResponseWithArgs
         |> required "args" decodeQueryArgs
 
-getIp : Http.Request (Http.Response (Response))
-getIp =
+getIp : (Result (Maybe (Http.Metadata, String), Http.Error) (Response) -> msg) -> Cmd msg
```

Comment:
Check in the code that is generated for tutorials that this is indeed the right signature. I guess these e2e tests didn't run for a while.